pkg/bundle/bundle.go

```diff
@@ -67,55 +67,85 @@ type Bundle struct {
 }
 
 // ReadConfig takes a bundle spec file as a string and maps it into the Bundle struct
-func ReadConfig(file string) (*BundleConfig, error) {
-	var body []byte
+func ReadConfig(file []string) (*BundleConfig, error) {
+	var tempBundleConfig struct {
+		Addons []*Bundle `yaml:"addons"`
+	}
+	bundleconfig := &BundleConfig{}
 
-	if file == "" {
-		defaultFile, err := getDefaultBundle()
-		if err != nil {
-			return nil, fmt.Errorf("failed retrieving default bundle file: %v", err)
-		}
-		body, err = io.ReadAll(defaultFile)
+	if len(file) == 0 {
+		files, err := defaultBundle.ReadDir("bundles")
 		if err != nil {
-			return nil, fmt.Errorf("unable to read default bundle file: %v", err)
+			fmt.Errorf("unable to process bundles: %v", err)
 		}
-	} else {
-		var err error
-		body, err = ioutil.ReadFile(file)
-		if err != nil {
-			return nil, fmt.Errorf("unable to read user provided file: %v", err)
-		}
-	}
 
-	if len(body) < 1 {
-		return nil, fmt.Errorf("file is empty")
-	}
+		for _, file := range files {
+			f, err := defaultBundle.ReadFile(filepath.Join("bundles", file.Name()))
+			if err != nil {
+				fmt.Printf("unable to read file: %v", err)
+				continue
+			}
 
-	bundleconfig := &BundleConfig{}
-	err := yaml.Unmarshal(body, bundleconfig)
-	if err != nil {
-		return nil, fmt.Errorf("unable to parse yaml file: %v", err)
-	}
-	return bundleconfig, nil
-}
+			if err := yaml.Unmarshal(f, &tempBundleConfig); err != nil {
+				fmt.Printf("Error unmarshaling file %s: %v\n", f, err)
```

Comment:
small tweaks to make this clearer to non-devs

```suggestion
				fmt.Printf("error parsing bundle file %s: %s\n", f, err.Error())
```